roxie/ccd/ccdactivities.cpp

```diff
@@ -588,6 +588,7 @@ class CRoxieAgentActivity : implements CInterfaceOf<IRoxieAgentActivity>, implem
     // The below are called on Roxie server and passed in context
     virtual unsigned getPriority() const { throwUnexpected(); }
     virtual char *getPlatform() { throwUnexpected(); }
+    virtual char *getDeploymentName() { throwUnexpected(); }
```

Comment:
Unlikely to be called, but I suspect cleaner to implement as a call like getDaliServers().

Reply:
agreed, I'll change.

Reply:
@ghalliday - have pushed that 1 line change (and squashed it in):
```
-    virtual char *getDeploymentName() { throwUnexpected(); }
+    virtual char *getDeploymentName() { return getDaliServers(); }
```